src/crispy/StrongHash.h

```diff
@@ -21,11 +21,27 @@
 #include <string_view>
 #include <type_traits>
 
+#ifdef __x86_64__
 #include <immintrin.h>
 
 #if defined(__AES__)
     #include <wmmintrin.h>
 #endif
+#elif __aarch64__
+#include "sse2neon.h"
```

Comment:
```suggestion
#include <crispy/sse2neon.h>
```

For the function implementations below, probably mark them `noexcept` and also mention were we've got them from. So that in case of upstream-improvements, it's easier to bugfix them.
I think the original code comes from here? https://github.com/JishinMaster/simd_utils/blob/master/sse2neon_wrapper.h

Reply:
I copied from here: https://github.com/f1ed/emp/blob/master/emp-tool/utils/block.h